contracts/payloads/IGP81/PayloadIGP81.sol

```diff
@@ -0,0 +1,503 @@
+    function action3() internal {
+        address cbBTC_USDT_DEX_ADDRESS = getDexAddress(22);
+        address cbBTC_USDT__cbBTC_USDT_VAULT_ADRESS = getVaultAddress(105);
+
+        {
+            // dust limits
+            Dex memory DEX_cbBTC_USDT = Dex({
+                dex: cbBTC_USDT_DEX_ADDRESS,
+                tokenA: cbBTC_ADDRESS,
+                tokenB: USDT_ADDRESS,
+                smartCollateral: true,
+                smartDebt: true,
+                baseWithdrawalLimitInUSD: 10_000, // $10k
+                baseBorrowLimitInUSD: 10_000, // $10k
+                maxBorrowLimitInUSD: 20_000 // $20k
+            });
+            setDexLimits(DEX_cbBTC_USDT); // Smart Collateral & Smart Debt
+
+            DEX_FACTORY.setDexAuth(cbBTC_USDT_DEX_ADDRESS, TEAM_MULTISIG, true);
+        }
+        
+        {// Set team multisig as vault auth for cbBTC_USDT T4 Vault
+            VAULT_FACTORY.setVaultAuth(
+                cbBTC_USDT__cbBTC_USDT_VAULT_ADRESS,
+                TEAM_MULTISIG,
+                true
+            );
+        }
+
+    }
+
+    // @notice Action 4:  Reduce max supply shares and max borrow limit for USDe-USDT<>USDT
+    function action4() internal {
+        address USDe_USDT_DEX_ADDRESS = getDexAddress(18);
+        address USDe_USDT__USDT_VAULT = getVaultAddress(93);
+        
+        {
+            // Dexcrease Max Supply Shares
+            IFluidDex(USDe_USDT_DEX_ADDRESS).updateMaxSupplyShares(
+                10_000_000 * 1e18 // 10M shares // $20M
+            );
+        }
+
+        {
+            // Update USDe-USDT<>USDT vault supply shares limit
+            IFluidAdminDex.UserSupplyConfig[]
+                memory config_ = new IFluidAdminDex.UserSupplyConfig[](1);
+            config_[0] = IFluidAdminDex.UserSupplyConfig({
+                user: USDe_USDT__USDT_VAULT,
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours, // 12 hours
+                baseWithdrawalLimit: 10_000_000 * 1e18 // 10M shares // $20M
+            });
+
+            IFluidDex(USDe_USDT_DEX_ADDRESS).updateUserSupplyConfigs(config_);
+        }
+
+        {
+            // [TYPE 2] USDe-USDT<>USDT | smart collateral & debt
+            Vault memory VAULT_USDe_USDT_USDT = Vault({
+                vault: USDe_USDT__USDT_VAULT,
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 0,
+                baseBorrowLimitInUSD: 9_000_000, // $9M
+                maxBorrowLimitInUSD: 18_000_000 // $18M
+            });
+
+            setVaultLimits(VAULT_USDe_USDT_USDT); // TYPE_2 => 93
+        }
+    }
+
+    // @notice Action 5:  Increase LTV & LT for USDe-USDT<>USDT
+    function action5() internal {
+        address USDe_USDT__USDT_VAULT = getVaultAddress(93);
+
+        uint256 CF = 92 * 1e2;
+        uint256 LT = 95 * 1e2;
```

Comment:
we should also increase LML, otherwise LT == LML 

@KaymasJain can you confirm the values here?